packages/connect-ui/src/index.tsx

```diff
@@ -85,11 +85,8 @@ export const ConnectUI = ({ postMessage, clearLegacyView }: ConnectUIProps) => {
         } else {
             // messages[0] could be null. in that case, legacy view is rendered
             switch (messages[0]?.type) {
-                case 'waiting-for-iframe-init':
-                    component = <Loader message="waiting for host to load" />;
-                    break;
-                case 'waiting-for-iframe-handshake':
-                    component = <Loader message="waiting for handshake from host" />;
```

Comment:
useless enumerating of all possible strings. lets just have 'loading' with some additional string info

Reply:
I agree, also those were too technical messages for user facing UI.